### ISSUE TITLE
Ganarg Analyzer drop table is wrong

### ISSUE THREAD

**Issue by Stump876** [azerothcore/azerothcore-wotlk#15766]
Original CC report https://github.com/chromiecraft/chromiecraft/issues/5295 confirmed on my local

### What client do you play on?

enGB

### Faction

Alliance

### Content Phase:

Generic

### Current Behaviour

No cloth, no gold, 1.47% chance on the [Schematic: Field Repair Bot 110G](https://wowgaming.altervista.org/aowow/?item=34114)

### Expected Blizzlike Behaviour

as demons they drop netherweave cloth and gold

For the Schematic, All the comments on wowhead speak of sub 50 kills needed, which would be quite rare at 1.47% drop chance
As usual with profession recipes it is aggregated from people that already have the recipe/don't have the profession

### Source

https://www.wowhead.com/wotlk/npc=23386/ganarg-analyzer#comments

### Steps to reproduce the problem

.setskill 202 1 75  
(engineering)
.go xyz 1928 7348 364 530
Kill a few thousand to get accurate statistics
(Just look at the DB)

### Extra Notes

https://wowgaming.altervista.org/aowow/?npc=23386

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/d030260dd4b7e0aae88f609222877bd2aabdafe0

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-bg-item-reward](https://github.com/azerothcore/mod-bg-item-reward)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-cta-switch](https://github.com/azerothcore/mod-cta-switch)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna](https://github.com/azerothcore/mod-eluna)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-low-level-rbg](https://github.com/azerothcore/mod-low-level-rbg)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-progression-system](https://github.com/azerothcore/mod-progression-system)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-rdf-expansion](https://github.com/azerothcore/mod-rdf-expansion)
- [mod-transmog](https://github.com/azerothcore/mod-transmog)
- [mod-weekend-xp](https://github.com/azerothcore/mod-weekend-xp)
- [mod-instanced-worldbosses](https://github.com/nyeriah/mod-instanced-worldbosses)
- [mod-zone-difficulty](https://github.com/azerothcore/mod-zone-difficulty)
- [mod-BreakingNewsOverride](https://github.com/AnchyDev/BreakingNewsOverride)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-exchange-npc](https://github.com/55Honey/Acore_ExchangeNpc)
- [lua-event-scripts](https://github.com/55Honey/Acore_eventScripts)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft

**Comment by elthehablo** [comment]
I agree about the probability. It's probably closer to 5-10%. Will check gold and netherweave issue on DB

**Comment by Tereneckla** [comment]
possibly also other things, but those 2 are very apparent
